vault/external_tests/identity/login_mfa_totp_test.go

```diff
@@ -254,6 +272,24 @@ func TestLoginMfaGenerateTOTPRoleTest(t *testing.T) {
 			t.Fatalf("MFA failed: %v", err)
 		}
 
+		if secret.Auth == nil || secret.Auth.ClientToken == "" {
```

Comment:
I'm a bit confused by these test changes - are they related to the new rest of the PR?  It looks like we're just checking in the audit log to see if the explicit write we did to sys/mfa/validate resulted in an audit record.

Reply:
yeah, I did not want to push another PR for it. I have made a note in the PR description about it.